UT4MasterServer.Web/src/types/rating.ts

```diff
@@ -2,6 +2,7 @@ export interface IRanking {
   rank: number;
```

Comment:
I didn't notice this eariler. Interface and filename don't match. Rename the file.

Reply:
All file names are named like this, no file name starts with I.

Reply:
i was talking about rating vs ranking name

Reply:
Rating is one thing, ranking is another. What exactly should I rename?

Reply:
Oh, I got what you meant. I will fix that.